packages/amplify_authenticator/lib/src/l10n/input_resolver.dart

```diff
@@ -364,9 +364,19 @@ class InputResolverKey {
       inputResolver.resolve(context, this);
 }
 
+/// The resolver class for input widgets.
+///
+/// This class is responsible for resolving text to display
+/// in input field titles, as well as hints, warnings and other
+/// guidance for the end-user.
 class InputResolver extends Resolver<InputResolverKey> {
   const InputResolver();
 
+  /// Returns the [String] that is displayed as the input title.
```

Comment:
General comment: no need to highlight types like this. The function signature will already have a link for all types. We generally want to highlight items which are not in the function declaration but may be in the surrounding scope, or related items.